localstripe/resources.py

```diff
@@ -3144,6 +3143,26 @@ def _calculate_amount_in_tier(self, quantity, index):
         t = self.plan.tiers[index]
         return int(t['unit_amount']) * quantity + int(t['flat_amount'])
 
+    @classmethod
+    def _api_list_all(cls, url, subscription=None, limit=None,
+                      starting_after=None, ending_before=None, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        try:
+            assert _type(subscription) is str
+            assert subscription.startswith('sub_')
+            Subscription._api_retrieve(subscription)
```

Comment:
Great to add this line!
But can it be moved outside the `try/catch`, I think there is no reason for this line to be in it.
Also can you add a small comment like `# to return 404 if not existant` on this line?